app/include/zmk/workqueue.h

```diff
@@ -0,0 +1 @@
+extern struct k_work_q lowprio_work_q;
```

Comment:
I don't *love* exposing this as an `extern` here. I'd vote for either a function (that would get inlined probably) or a C-macro/define at least.

In particular, if we have a space constrained device (RAM wise) we might want to just-reuse the system workqueue instead of adding a second queue, and using the define/function will let use swap those out w/o the consumer caring as much.

Thoughts?

Reply:
I've introduced `zmk_lowprio_work_q()` but generally I'm hesitant to make things like this configurable unless if there's a good concrete reason - ie 1KB is the difference between fitting and not fitting on a chip.

Furthermore currently the purpose of the low priority workqueue is not well defined. It may end up being used for other tasks that must run with a lower priority than the system workqueue on low resource MCUs, similar to how battery notification is for BLE.